src/RazorSdk/Tasks/Microsoft.NET.Sdk.Razor.Tasks.csproj

```diff
@@ -0,0 +1,118 @@
+<Project>
+
+  <PropertyGroup>
+    <RazorSdkRoot>$(RepoRoot)\src\RazorSdk\</RazorSdkRoot>
+    <PackageId>Microsoft.NET.Sdk.Razor</PackageId>
+    <OutDirName>$(Configuration)\Sdks\$(PackageId)\tasks</OutDirName>
+  </PropertyGroup>
+
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+
+    <NoPackageAnalysis>true</NoPackageAnalysis>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
+    <!-- MSBuild Task DLLs need to be versioned with every build -->
+    <AutoGenerateAssemblyVersion>true</AutoGenerateAssemblyVersion>
+
+    <OutputPath>$(BaseOutputPath)</OutputPath>
+    <IsPackable>true</IsPackable>
```

Comment:
I don't think you need to pack these?

Reply:
I think it is needed based on the way the generate layout targets work. Both the Web SDK and Publish SDK have it set.